zabbix_smartmontools/__init__.py

```diff
@@ -400,3 +404,31 @@ def parseConfig(path=None):
         d['Disks'] = ["%s %s" % (k, v) for (k, v) in config['Disks'].items()]
 
     return d
+
+def main():
+    fail_ifNot_Py3()
+
+    cmd = sys.argv[1]
+    host = '"%s"' % (sys.argv[2])
+    config = parseConfig()
+
+    configError = None
+    if not 'Disks' in config:
+        scanDisks_Out = scanDisks(config, cmd)   # scan the disks
+
+        configError = scanDisks_Out[0]   # SCAN_OS_NOCMD, SCAN_OS_ERROR, SCAN_UNKNOWN_ERROR
+        diskList = scanDisks_Out[1]
+    else:
+        diskList = config['Disks']
+
+    jsonData, senderData = getAllDisks(config, host, cmd, diskList)
+
+    if configError:
+        senderData.append('%s smartctl.info[ConfigStatus] "%s"' % (host, configError))
+    elif not diskList:
+        senderData.append('%s smartctl.info[ConfigStatus] "NODISKS"' % (host))   # if no disks were found
+    else:
+        senderData.append('%s smartctl.info[ConfigStatus] "CONFIGURED"' % (host))   # signals that client host is configured (also fallback)
+
+    link = r'https://github.com/nobodysu/zabbix-smartmontools/issues'
+    processData(senderData, jsonData, config['agentConf'], config['senderPath'], host, link)
```

Comment:
```suggestion
    processData(senderData, jsonData, config['agentConf'], config['senderPath'], host)
```
Not relevant anymore.